# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libgda/recent-bdb.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Fix compilation for recent 4.3 bdb versions ...

  - Rene Rebe <rene@exactcode.de>

--- libgda-1.3.1-orig/providers/bdb/gda-bdb-recordset.c	2005-01-10 13:29:38.000000000 +0100
+++ libgda-1.3.1/providers/bdb/gda-bdb-recordset.c	2005-04-06 09:39:36.667078360 +0200
@@ -228,7 +228,7 @@
 	/* get the number of records in the database */
 	ret = dbp->stat (dbp,
 			 &statp,
-#if BDB_VERSION < 40000
+#if BDB_VERSION < 40000 ||  BDB_VERSION > 40300
 			 NULL,
 #endif
 			 0);
